pkg/env/prod.go

```diff
@@ -132,7 +132,7 @@ func newProd(ctx context.Context, log *logrus.Entry) (*prod, error) {
 		}
 		p.acrDomain = acrResource.ResourceName + "." + p.Environment().ContainerRegistryDNSSuffix
 	} else {
-		p.acrDomain = "arointsvc" + "." + p.Environment().ContainerRegistryDNSSuffix
+		p.acrDomain = "arointsvc" + "." + azure.PublicCloud.ContainerRegistryDNSSuffix // TODO: should this be cloud aware?
```

Comment:
doesn't exist yet in MAG?

Reply:
not yet - needs to be hardcoded to PublicCloud for now. we can revisit it in the future.